assets/training/model_management/environments/model-management/context/requirements.txt

```diff
@@ -1,7 +1,7 @@
 azure-core=={{latest-pypi-version}}
 azure-ai-ml=={{latest-pypi-version}}
 azure-identity=={{latest-pypi-version}}
-azureml-evaluate-mlflow=={{latest-pypi-version}}
+azureml-evaluate-mlflow~={{latest-pypi-version}}
```

Comment:
May be do this for all azureml packages. azureml-automl-core, azureml-telemetry?